### ISSUE TITLE
Nothing happens?

### ISSUE THREAD

**Issue by nickbarrow** [steventheworker/Dock-Expos--home#2]
Not sure if I'm missing something here but this app doesn't do anything for me. I watched both videos on the website and enabled access to Accessibility and Input Monitoring and still nothing. How is this app meant to work? Am I supposed to hit a hotkey when hovering or should the previews just show up when hovering dock icons?

MacBook Pro 16" 2021, running Monterey on M1 Pro chip, thanks

**Comment by steventheworker** [comment]
If you ran Dock Exposé before giving permissions, try restarting Dock Exposé.
If it still doesn't work reinstall by removing permissions from Accessibility and Input Monitoring, then restart the mac and try running it after giving Accessibility and Input Monitoring.

Previews should show up by hovering on the app icon, without any other user input.
